taskcluster/scripts/write-dummy-secret.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import argparse
+import errno
+import json
+import os
+
+
+def write_secret_to_file(path, secret, key, json_secret=False):
+    path = os.path.abspath(os.path.join(os.path.dirname(__file__), '../../' + path))
+    try:
+        os.makedirs(os.path.dirname(path))
+    except OSError as error:
+        if error.errno != errno.EEXIST:
+            raise
+    print("Outputting secret to: {}".format(path))
+
+    with open(path, 'w') as f:
+        if json_secret:
+            secret = json.dumps(secret)
```

Comment:
Question: out of curiosity, what values do we expect for  `secret` that need encoding here?

Reply:
Good question! I know the firebase token is json https://github.com/mozilla-mobile/reference-browser/blob/05064951977d8f5c8d4fdc542e7de9d399708fdb/taskcluster/ci/test/kind.yml#L61, for instance.